github/secret_scanning.go

```diff
@@ -120,6 +120,52 @@ type SecretScanningAlertUpdateOptions struct {
 	ResolutionComment *string `json:"resolution_comment,omitempty"`
 }
 
+// PushProtectionBypassRequest represents the parameters for CreatePushProtectionBypass.
+type PushProtectionBypassRequest struct {
+	// Reason provides a justification for the push protection bypass.
```

Comment:
Could we enhance the comment from GitHub's docs?
```suggestion
	// The reason for bypassing push protection.
	// Can be one of: false_positive, used_in_tests, will_fix_later
```